src/tests/CMakeLists.txt

```diff
@@ -0,0 +1,53 @@
+add_option(-mmmx)
+add_option(-msse)
+add_option(-msse2)
+add_option(-msse3)
+
+add_option(-mstackrealign)
+
+include(UsewxWidgets)
+include_directories(${CMAKE_BINARY_DIR}/src/core/go_defs.h ${CMAKE_SOURCE_DIR}/src/core)
+
+# Add here your tests files
+set(go_test_lib
+    # Test framework - Don't modify
+    GOTestResult.cpp
```

Comment:
Please distribute test classes for certain GO classes (ex GOTestSwitch) across subdirectories

Reply:
Yes I know but don't know how to make it correctly through cmakelists

Reply:
> Yes I know but don't know how to make it correctly through cmakelists

``
set(go_test_lib
    dir1/GOTest1.cpp
    dir2/dir3/GOTest2.cpp
)
``